skimage/measure/tests/test_regionprops.py

```diff
@@ -693,3 +695,24 @@ def test_extra_properties_table():
                             )
     assert_array_almost_equal(out['median_intensity'], array([2., 4.]))
     assert_array_equal(out['pixelcount'], array([10, 2]))
+
+
+def test_multichannel():
+    """Test that computing multichannel properties works."""
+    astro = data.astronaut()
```

Comment:
can you please downscale or crop a bit the image? `slic` takes 0.3s on my machine to execute on the whole image, not a big deal but our test suite does take a long time to execute so let's try to keep tests really fast if possible.